models/stg_amazon_ads__ad_group_history.sql

```diff
@@ -21,12 +21,12 @@ fields as (
 final as (
     
     select 
-        _fivetran_synced,
+        _fivetran_synced, 
         campaign_id,
         creation_date,
         default_bid,
         id as ad_group_id,
-        last_updated_date,
+        last_updated_date, 
```

Comment:
In downstream models we will want to filter for only the latest version of the history. Can you add an `is_msot_recent_record` boolean field to these history models. 

See [here](https://github.com/fivetran/dbt_apple_search_ads_source/blob/db9bb313686b4ea19e3c2b41e6df4cb62917862e/models/stg_apple_search_ads__campaign_history.sql#L32) for an example.

Reply:
Be sure to make the same updates across all the history staging models.

Reply:
done.